### PR TITLE
RHBPMS-5084: Process definition list in start process instance dialog…

### PR DESCRIPTION
**Pull request by nmirasch**
… incomplete

### CODE REVIEW

**Comment by nmirasch** [comment]
@cristianonicolai, @bsremac could you take a look? Thanks!

File: jbpm-console-ng-process-runtime-forms/jbpm-console-ng-process-runtime-forms-client/src/main/java/org/jbpm/console/ng/pr/forms/client/editors/quicknewinstance/QuickNewProcessInstancePopup.java [hunk 1]
```diff
@@ -72,6 +71,9 @@
 
     }
 
+    public static final int PAGE_SIZE_UNLIMITED = 1000;
```
Comment (cristianonicolai) [inline jbpm-console-ng-process-runtime-forms/jbpm-console-ng-process-runtime-forms-client/src/main/java/org/jbpm/console/ng/pr/forms/client/editors/quicknewinstance/QuickNewProcessInstancePopup.java]: I think for this sort of cases we end up using Integer.MAX_VALUE as the limit.

File: jbpm-console-ng-process-runtime-forms/jbpm-console-ng-process-runtime-forms-client/src/main/java/org/jbpm/console/ng/pr/forms/client/editors/quicknewinstance/QuickNewProcessInstancePopup.java [hunk 2]
```diff
@@ -178,11 +180,11 @@ protected void loadFormValues() {
...
         processDeploymentIdListBox.clear();
-        currentFilter = new PortableQueryFilter( 0,
-                                                 10,
-                                                 false, "",
-                                                 "",
-                                                 true );
+        currentFilter = new PortableQueryFilter(0,
+                                                PAGE_SIZE_UNLIMITED,
+                                                false, "",
+                                                FIELD_ID_PROCESSNAME,
```
Comment (cristianonicolai) [inline jbpm-console-ng-process-runtime-forms/jbpm-console-ng-process-runtime-forms-client/src/main/java/org/jbpm/console/ng/pr/forms/client/editors/quicknewinstance/QuickNewProcessInstancePopup.java]: I think this sort is not in place on master, would be good to send this PR in there as well.

**Comment by nmirasch** [comment]
@cristianonicolai filter changed to use Integer.MAX_VALUE as a page size.
Generating the PR with this changes to master right now.

**Pull request merged**